packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -121,6 +121,7 @@ function SquadPostContent({
           <PostSourceInfo
             date={postDateFormat(post.createdAt)}
             source={post.source}
+            className="!typo-body"
```

Comment:
I see we are adding more and more important overrides, not a big fan of this trend.
Perhaps we should rather evaluate other rendering options then?